listener/__tests__/ip-addresses.test.js

```diff
@@ -23,6 +23,11 @@ describe('ip addresses', () => {
     jest.resetModules();
   });
 
+  afterEach(async (done) => {
```

Comment:
I think you can remove `done` call if you use `async`

Reply:
I will do a new commit removing every done call in functions using async